### ISSUE TITLE
Narrator: Focus rectangle for treeNode is not in the correct position after expanded/collapsed the non-first parent treeNode when treeView has multiple level treeNode

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#7056]
### .NET version

7.0.100-preview.4.22218.7

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No, it is not a regression issue

### Issue description

Narrator's focus rectangle for treenode is not in the correct position after expanded/collapsed the non-first parent treeNode when treeView has multiple level treeNode, and found that it always repro in TreeView with small size

### Steps to reproduce

1. Open following testing app
[WinFormsApp42.zip](https://github.com/dotnet/winforms/files/8519775/WinFormsApp42.zip)
2. Build and run app
3. Launch Narrator, and press left/right arrow to collapse/expand non-first parent treeNode

Result: Narrator's focus rectangle for treenode is not in the correct position
![image](https://user-images.githubusercontent.com/26474449/164187733-267c2433-5743-422a-a893-94de5f9c75dc.png)
![image](https://user-images.githubusercontent.com/26474449/164187748-1e4c4eb6-9902-4537-a683-dbe3603cf6ea.png)
![image](https://user-images.githubusercontent.com/26474449/164187777-750b1b6e-ce99-4741-aa32-7b13605f09aa.png)

**Comment by dkazennov** [comment]
The issue with the Narrator focus doesn't reproduce in WinformsControlTest TreeView test, both with changes made by PR #7175 and with the TreeView test locally replaced with the form from the testing application. The issue doesn't reproduce with checkout to older project commits e.g. v7.0.0-preview.2.22152.2.

The issue reproduces in the separate testing application, both with dotnet release (7.0.100-preview.3.22179.4) and with most recent compiled .dll files from the project. No changes tested locally (including changes made to Bounds field of TreeNode, to navigation between Accessible Objects and to TreeView.WmNotify() method) fixes the issue. According to Inspect and Accevent right nodes have IsSelected and HasFocus properties.

**Comment by dkazennov** [comment]
An additional testing outside of Winforms shows the same Narrator highlight behavior with scrollbars in different applications including Visual Studio and in Windows Explorer.

![image](https://user-images.githubusercontent.com/102961955/168862604-bb9fe8af-f90a-423e-b505-2c2c6a58d7ed.png)

**Comment by Tanya-Solyanik** [comment]
Great investigation! Yeah, looks like we need to raise some sort of [accessibility event](https://github.com/dotnet/winforms/blob/d8016d116e11d476adb1950bfdc09dc2150b90f0/src/System.Windows.Forms/src/System/Windows/Forms/AccessibleEvents.cs#L308). But since this does not work in File explorer either, and new acc event might be not expected and breaking for automation tests, we can "won't fix" this bug and wait until we have customer reports.
